plugins/UM3NetworkPrinting/resources/qml/ConfigurationChangeBlock.qml

```diff
@@ -182,11 +185,13 @@ Item {
                     }
                     text: catalog.i18nc("@label", "Override");
                     visible: {
-                        var length = root.job.configurationChanges.length;
-                        for (var i = 0; i < length; i++) {
-                            var typeOfChange = root.job.configurationChanges[i].typeOfChange;
-                            if (typeOfChange === "material_insert" || typeOfChange === "buildplate_change") {
-                                return false;
+                        if (root.job & root.job.configurationChanges) {
```

Comment:
Shouldn't this be (root.job && root.job.configurationChanges) ?

Reply:
Yup good catch thanks